drivers/include/mfrc522.h

```diff
@@ -0,0 +1,844 @@
+                                             uint8_t *buffer_atqa,
+                                             uint8_t *buffer_size);
+
+/**
+ * @brief Transmits WUPA, Type A. Invites PICCs in state IDLE and HALT to go to
+ *        READY(*) and prepare for anti-collision or selection. 7 bit frame.
+ *
+ * @note  When two PICCs are in the field at the same time we often get
+ *        MFRC522_STATUS_TIMEOUT - probably due do bad antenna design.
+ *
+ * @param[in]      dev          Device descriptor of the MFRC522
+ * @param[out]     buffer_atqa  Buffer to store the ATQA in
+ * @param[inout]   buffer_size  Buffer size, at least two bytes. Also number of
+ *                              bytes returned if MFRC522_STATUS_OK.
+ *
+ * @return MFRC522_STATUS_OK on success, MFRC522_STATUS_??? otherwise
+ */
+mfrc522_status_code_t mfrc522_picc_wakeup_a(mfrc522_t *dev,
+                                            uint8_t *buffer_atqa,
+                                            uint8_t *buffer_size);
+
+/**
+ * @brief Transmits REQA (Type A) or WUPA (Type A) commands
+ *
+ * @note  When two PICCs are in the field at the same time we often get
+ *        MFRC522_STATUS_TIMEOUT - probably due do bad antenna design.
+ *
+ * @param[in]     dev          Device descriptor of the MFRC522
+ * @param[in]     command      Command to send - MFRC522_PICC_CMD_ISO_14443_REQA
+ *                             or MFRC522_PICC_CMD_ISO_14443_WUPA
+ * @param[out]    buffer_atqa  Buffer to store the ATQA in
+ * @param[inout]  buffer_size  Buffer size, at least two bytes. Also number of
+ *                             bytes returned if MFRC522_STATUS_OK.
+ *
+ * @return MFRC522_STATUS_OK on success
+ * @return MFRC522_STATUS_INVALID if command was neither
+ *         MFRC522_PICC_CMD_ISO_14443_REQA nor MFRC522_PICC_CMD_ISO_14443_WUPA,
+ *         MFRC522_STATUS_??? otherwise
+ */
+mfrc522_status_code_t mfrc522_picc_reqa_or_wupa(mfrc522_t *dev,
+                                                mfrc522_picc_command_t command,
+                                                uint8_t *buffer_atqa,
+                                                uint8_t *buffer_size);
+
+/**
+ * @brief Transmits SELECT/ANTICOLLISION commands to select a single PICC
+ *
+ * @note  Before calling this function the PICCs must be placed in the READY(*)
+ *        state by calling mfrc522_picc_request_a() or mfrc522_picc_wakeup_a().
+ *        On success:
+ *          - The chosen PICC is in state ACTIVE(*) and all other PICCs have
+ *            returned to state IDLE/HALT. (Figure 7 of the ISO/IEC 14443-3 draft.)
+ *          - The UID size and value of the chosen PICC is returned in *uid along
+ *            with the SAK.
+ *
+ * A PICC UID consists of 4, 7 or 10 bytes.
+ * Only 4 bytes can be specified in a SELECT command, so for the longer UIDs two
+ * or three iterations are used:
+ * 		UID size	Number of UID bytes		Cascade levels		Example of PICC
```

Comment:
Please don't use tabs for indent. I recommend to configure your editor to match the coding convention. E.g. vim / neovim can configure such settings based on paths, so that all files in RIOT could be set to 4 spaces per level of indent.

Reply:
Apparently I forgot to replace them after copying the doc from the original arduino driver. Replaced by spaces now.